src/Plugin.php

```diff
@@ -26,6 +26,13 @@ class Plugin {
    */
   const L10N = self::PREFIX;
 
+  /**
+   * Product GTIN custom meta field name.
+   *
+   * @var string
+   */
+  const GTIN_CUSTOM_FIELD_NAME = '_shop-standards_gtin';
```

Comment:
Why did you decide to add this as a single constant instead of writing it statically in the function? 🤔